CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+CHANGES IN VERSION 1.14.10 (FROM 1.13.0)
+===================================
+
+ Appium 1.14.0 is a minor release.
+
+ #### General
+* Minimum Node version is bumped up to v10
+* The default driver for Android is now set to [UiAutomator2](https://github.com/appium/appium-uiautomator2-driver). If the [UiAutomator1](https://github.com/appium/appium-android-driver) driver is desired, then this can be achived by setting `automationName=UiAutomator1` in the capabilities
+
+ #### Android (Espresso)
+* Fix problems with activity startup not working when the package name value in `appActivity` is different from the one in `appPackage` [#441](https://github.com/appium/appium-espresso-driver/pull/441)
+
+ #### iOS (XCUITest)
+* Make `platformVersion` a required capability for iOS Simulators [#954](https://github.com/appium/appium-xcuitest-driver/pull/954)
+* Enforce Simulator shutdown if `resetOnSessionStartOnly` is set to false [#950](https://github.com/appium/appium-xcuitest-driver/pull/950)
+* Fixed the issue with [addresses problems with long startup times for Safari WebViews on iOS 12.2](https://github.com/appium/appium/issues/12590)
+
```

Comment:
I can remember below: (after 1.3.0)

## general
- Support _execute driver_ command to run a WebdriverIO script against the current session https://github.com/appium/appium/pull/12807 . It requires client update.
- add `--allow-insecure` and `--deny-insecure` server arguments to control 
control access to features https://github.com/appium/appium/pull/12778 , https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/security.md
- Drop `--enable-heapdump` for debugging https://github.com/appium/appium/pull/12609
- Fix `--tmp` server arg https://github.com/appium/appium/pull/12585
- Can get selected image by `find element by image` for debugging https://github.com/appium/appium-base-driver/pull/327


## iOS(xcuitest)
- Support Xcode 11 https://github.com/appium/WebDriverAgent/pull/171
- Update to call `idb` instead of `fbsimctl` which is used for some commands for simulator environment https://github.com/appium/appium/pull/12574/files
- Configure keyboard preference https://github.com/appium/appium/pull/12737
    - Disable `Auto-Correction` and `Predictive` in `Keyboards` preference by default
    - Disable keyboard tutorial for iOS 13
    - Force toggle software keyboard on for simulator.
- Fix file translation for real device.
    - https://github.com/appium/appium/pull/12710
    - Read https://github.com/appium/appium/blob/master/docs/en/writing-running-appium/ios/ios-xctest-file-movement.md about the rule
- Fix WDA manual code signing issue which happens in some cases https://github.com/appium/appium-xcuitest-driver/pull/961
- Fix WDA build issue in Python 3 environment https://github.com/appium/WebDriverAgent/pull/168

Reply:
> Support Xcode 11 appium/WebDriverAgent#171

This is not tagged yet, so not available

Reply:
one more change for iOS:
- `platformVersion` capability is now mandatory for Simulators and optional for real devices (but only if Appium can determine the version from ideviceinfo output)

Reply:
Done. Thanks